src/renderer/component/copyableText/view.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react';
+import { clipboard } from 'electron';
+import { FormRow } from 'component/common/form';
+import Button from 'component/button';
+import * as icons from 'constants/icons';
+
+type Props = {
+  copyable: string,
+  doNotify: ({ message: string, displayType: Array<string> }) => void,
+};
+
+export default class CopyableInput extends React.PureComponent<Props> {
+  constructor() {
+    super();
+
+    this.input = null;
+  }
+
+  input: ?HTMLInputElement;
+
+  render() {
+    const { copyable, doNotify } = this.props;
+
+    return (
+      <FormRow verticallyCentered padded stretch>
+        <input
+          className="input-copyable form-field__input"
+          readOnly
+          value={copyable || ''}
+          ref={input => {
+            this.input = input;
+          }}
+          onFocus={() => {
+            if (this.input) {
+              this.input.select();
+            }
+          }}
+        />
+        <Button
+          noPadding
+          button="secondary"
+          icon={icons.CLIPBOARD}
+          onClick={() => {
+            clipboard.writeText(copyable);
+            doNotify({
+              message: __('Text copied'),

```

Comment:
We can default to `Text copied` but we should be able to pass in any message to use. Ex: on the wallet overview/shapeshift, we should say `Address copied`.